sway/tree/output.c

```diff
@@ -225,6 +225,10 @@ void output_disable(struct sway_output *output) {
 
 	root_for_each_container(untrack_output, output);
 
+	if (output->bg_pid) {
+		terminate_swaybg(output->bg_pid);
```

Comment:
I think it's important to reset `output->bg_pid` here in case the output is enabled again.